ansible/roles/xsnippet_api/templates/caddy.j2

```diff
@@ -0,0 +1,28 @@
+(cors) {
+  @cors_preflight method OPTIONS
+  @cors header Origin {args.0}
+
+  handle @cors_preflight {
+    header Access-Control-Allow-Origin "{args.0}"
+    header Access-Control-Allow-Methods "GET, POST, PUT, PATCH, DELETE"
+    header Access-Control-Allow-Headers "Content-Type"
+    # header Access-Control-Max-Age "3600"
```

Comment:
What's up with this header?

Reply:
Ops. I turned it off so my browser won't cache the preflight request.